test/check-machines-nics

```diff
@@ -85,6 +82,34 @@ class TestMachinesNICs(VirtualMachinesCase):
         b.wait_in_text("#vm-subVmTest1-network-3-source", "virbr0")
         b.wait_in_text("#vm-subVmTest1-network-3-ipv4-address", "192.168.122.")
 
+        # Check Plug and unplug NICs
+        def checkPlugAndUnplugNICs():
+            mac = b.text("#vm-subVmTest1-network-1-mac")
+
+            b.wait_in_text("#vm-subVmTest1-network-1-state", "up")
+            b.wait_in_text("#delete-vm-subVmTest1-iface-1 + button", "Unplug")
+
+            # Click "Unplug"
+            b.click("#delete-vm-subVmTest1-iface-1 + button")
```

Comment:
That's really weird way to select the unplug button. You essentially say - take the one after the delete button. 
Please add in the JS an id for the unplug/plug buttons and use that instead.

Reply:
OK, let me add the id in another PR